dev/test/wdio.conf.js

```diff
@@ -1,4 +1,6 @@
 exports.config = {
+  port: '9515',
```

Comment:
Is this standard webdriver port, or how did we choose this? Thought worth a comment perhaps.